module/config.js

```diff
@@ -106,5 +106,61 @@ export default class WorldAnvilConfig extends FormApplication {
       config: true
     });
 
+    // Add the customizable labels for each importable page
+    //-------------------
+    game.settings.register("world-anvil", "mainArticlePage", {
```

Comment:
These settings feel like a level of customization that may not be necessary to me - I wonder if we could keep it simpler at first and see what feedback we get from users?

Reply:
See comment bellow.

The parameters are far from being mandatory. An good alternative could be to create translation file for every language. Or ate the least put how to create on with the necessary data inside the Readme file.